packages/generator/src/v2.0/generateClientSdkVersionTwoPointZero.ts

```diff
@@ -30,6 +30,7 @@ export async function generateClientSdkVersionTwoPointZero(
   fs: MinimalFs,
   outDir: string,
   packageType: "module" | "commonjs" = "commonjs",
+  additionalUserAgents: string[] = [],
```

Comment:
This is unused for 2.0 as foundry-sdk-generator doesn't allow generating 2.0 yet